CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # nf-core/tools: Changelog
 
+## [v1.13.3 - Copper Crocodile Resurrection :crocodile:](https://github.com/nf-core/tools/releases/tag/1.13.2) - [2021-03-24]
+
+* Running tests twice with `nf-core modules create-test-yml` to catch unreproducible md5 sums [[#890](https://github.com/nf-core/tools/issues/890)]
+* Fix sync error again where the Nextflow edge release needs to be used for some pipelines
+* Fix bug with `nf-core lint --release` (`NameError: name 'os' is not defined`)
+
```

Comment:
```suggestion
* Fix bug with `nf-core lint --release` (`NameError: name 'os' is not defined`)
* Added linebreak to linting comment so that markdown header renders on PR comment properly
* In moule create command - if no bioconda package is found, prompt user for a different bioconda package name

```